x-pack/plugins/infra/public/components/subscription_splash_content.tsx

```diff
@@ -103,16 +102,62 @@ export const SubscriptionSplashPrompt: React.FC = () => {
     );
   }
 
+  return (
+    <PageTemplate {...templateProps} isEmptyState={true}>
```

Comment:
This is not a new change - the old version of the prompt is moved as part of the `SubscriptionSplashPage` component and the new Inventory version is now `SubscriptionSplashPrompt`